lib/Test/Perl/Critic.pm

```diff
@@ -47,6 +47,15 @@ sub import {
 
 #---------------------------------------------------------------------------
 
+{
+
+my $pc;
+my $critic = sub {
+    my ( %args ) = @_;
```

Comment:
Maybe it would be wise to singletonize the %args too, so that you don't get differently inited Critic instance after calling this with arguments different from the ones in the first call.

Reply:
@Dominik-Sauer I haven't exactly singletonize %args, but rather reset the possibly initialized P::C object with each call to `import`. I guess that solves the issue as well.